src/types.rs

```diff
@@ -64,6 +64,12 @@ impl PartialEq<crate::interface::MappingType> for AstarteType {
             };
         }
 
+        if other == &crate::interface::MappingType::LongInteger {
```

Comment:
Specify at top of function the `use` declaration for better readability.
``` Rust
use crate::interface::MappingType 
``` 
....
``` Rust
if other == &MappingType::LongInteger
```